src/behavior.ts

```diff
@@ -189,6 +239,18 @@ export class SinkBehavior<A> extends ProducerBehavior<A> {
   constructor(public last: A) {
     super();
   }
+  publish(a: A) {
+    if (this.last !== a) {
+      const t = tick();
+      this.last = a;
+      this.changedAt = t;
+      this.pulledAt = t;
+      this.pushToChildren(t);
+    }
+  }
+  push(a: A) {
```

Comment:
I don't think the complexity of an alias is warranted. I think we should decide on either `push` or `publish` and remove the alias.